### PR TITLE
new pkg: cabextract

### PR DESCRIPTION
**Pull request by tuananh**
use case:
- some application use pdf rendering engine, which requires use of msttcorefonts, which then need cabextract to extract the fonts.
- this pkg is required in order for the msttcorefonts installer to work. will submit a subsequent PR later for that.

Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

#### For new version streams

- [ ] The upstream project actually supports multiple concurrent versions.
- [ ] Any subpackages include the version string in their package name (e.g. `name: ${{package.name}}-compat`)
- [ ] The package (and subpackages) `provides:` logical unversioned forms of the package (e.g. `nodejs`, `nodejs-lts`)

#### For security-related PRs

- [ ] The security fix is recorded in the [advisories](https://github.com/wolfi-dev/advisories) repo

#### For version bump PRs

- [ ] The `epoch` field is reset to 0

#### For PRs that add patches

- [ ] Patch source is documented

### CODE REVIEW

**Comment by tuananh** [comment]
@ajayk im not sure what lint rule is failing here. Can you help?

**Comment by ajayk** [comment]
> @ajayk im not sure what lint rule is failing here. Can you help?

Not sure what's happening here , I copied the changes from here https://github.com/wolfi-dev/os/pull/7423/files which I used as a test it fails here , but passes in the test

File: cabextract.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - libmspack-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://www.cabextract.org.uk/cabextract-${{package.version}}.tar.gz
```
Comment (rawlingsj) [inline cabextract.yaml:20]: I think we can switch this to `git-checkout` https://github.com/kyz/libmspack

File: cabextract.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+         --with-external-libmspack
+
+  - uses: autoconf/make
+
+  - uses: autoconf/make-install
+
+  - uses: strip
+
+update:
+  enabled: false
```
Comment (rawlingsj) [inline cabextract.yaml:39]: As the source is on GitHub we can track the tags https://github.com/kyz/libmspack/tags

**Review by rawlingsj — CHANGES_REQUESTED** [review]
Hi @tuananh - it looks like we can switch this to get the sources direct from GitHub and also enable auto updates.  I'm happy to do that, I can push to your branch.

File: cabextract.yaml (hunk 1 above)
Comment (rawlingsj) [reply cabextract.yaml:20]: hmm that didn't work as I suspect the upstream release is adding extra configure files.  Let's ignore this comment.

**Review by rawlingsj — APPROVED** [review]

**Pull request merged**